daemon/config.go

```diff
@@ -124,7 +124,7 @@ func (c *Config) SetKVBackend() error {
 	case kvstore.Etcd:
 		log.Infof("using etcd as key-value store")
 		c.EtcdConfig = &etcdAPI.Config{}
-		c.EtcdConfig.Endpoints = []string{"http://127.0.0.1:4002"}
+		c.EtcdConfig.Endpoints = []string{"http://127.0.0.1:4001"}
```

Comment:
Actually this should not be changed. This function is only used for unit tests, and the port `4002` is set up in `Makefile:22`